CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+
+* Updated WDL crate to latest (by pointing to `wdl:main`
+  as the latest crates haven't been released yet).
```

Comment:
Should this be the default for `sprocket:main` and the release process for `sprocket` will require pinning the `wdl` crate versions?

Reply:
That's a good question. I think the our current procedure is to really only touch this repo directly before/after/during a `wdl` crate release, so we've always had `sprocket:main` point to official releases. As the `sprocket` binary and `wdl` crates mature, we may want to have a more synchronous development flow.

So I'm not sure. @peterhuene I'm curious if you have any thoughts on this Q.

Reply:
This was addressed by @claymcleod over Slack. The decision made was that `sprocket:main` should always point to official `wdl` releases.